components/Trombinoscope.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
How is this component different from `BuilderTeam.vue` ?

Reply:
Ref : https://github.com/BDE-Polytech-MTP/website-front/pull/58#pullrequestreview-746038770

I think it was because `BuilderTeam` is not specified in `Builder`, preventing it from being used in `website.json`.  If it is a duplicate, maybe we should consider adding a `BuilderTeam` "constructor" in `Builder` (*if the component is not incompatible with being built from it*).

Reply:
I think we should also consider splitting up website.json, it becomes too large to be easilty readable. But, yeah, BuildTeam and this PR seem duplicates, we should remove one of them. It let @Alexandre-Fernique (this PR) and @julien-vendran (BuilderTeam) discuss which of them is more suitable for the usage.

We should keep in mind the trombinoscope won't be integrated in an other page than the team page (in my opinion). So this being not accessible from builder isn't a problem. A simple `team.json` file should be sufficient

Reply:
In addition to splitting the team from `website.json` into a separate file, we should do the same for the BDS and BDA clubs as they might become too cumbersome.

@QuentinDesbrousses is the one working on https://github.com/BDE-Polytech-MTP/website-front/tree/quentin/page-bds and the future `page-bda` branch. This idea was considered but not implemented, it might be a good feature after all. I'll let him decide.

Reply:
Yes splitting the file website.json is a good idea but not essential at the moment. @BrokenSwing I think we need to build different trombinoscope for bde, bda and bds because they want their trombinoscope on their pages.

Reply:
their own pages*